pkg/controllers/customroute/custom_route_controller.go

```diff
@@ -289,3 +312,46 @@ func (c *customRouteController) getFieldManager() string {
 	// TODO find a way to get the client name and combine it with the controller name automatically
 	return "AuthenticationCustomRouteController"
 }
+
+func (c *customRouteController) removeOperands(ctx context.Context, ingressConfig *configv1.Ingress, secretName string) error {
```

Comment:
secretName is unused.

Reply:
Fixed with 4200edde03363dde52a1473266ce0f0b22be825d